templates/serviceForm.html

```diff
@@ -183,3 +188,136 @@ <h3 id="titleH3"></h3>
 </form>
 
 <script src="{{ url_for('static', filename='serviceForm.js') }}"></script>
+<script>
+
+    // Note: logic needs to be able to access python API and hence was placed here
```

Comment:
Well, all you're doing is injecting the `url_for`... 
I'm always a bit iffy about injecting Jinja stuff into JavaScript, generally speaking it's insecure; if an attacker manages to get the server to send something nasty then it will execute as JavaScript on a client's browser. 

You could hardcode the URL, but maybe a better way to pass data from the Jinja2 backend to the JS would be something like
```html
<input id="myUrl" type="hidden" value="{{ url_for(...) }}">
<script> // put this in a separate file
const myUrl = document.getElementById("myUrl").value;
</script>
```
You could alternatively use the `data-...` attributes.

Reply:
Okay, thanks. I've seen something similar in the pewSheet.html, which then would need fixing as well? What do you mean by the data-... attributes?

Reply:
> data-... attributes

e.g.
```html
<p id="foo" data-url="{{ url_for(...) }}"></p>
<script>
const myUrl = document.getElementById("foo").dataset.url
</script>